client/elements/text/sc-text-page-selector.html

```diff
@@ -281,21 +281,19 @@ <h3>[[lastError.statusText]]</h3>
               if (!responseData) {
                   return;
               }
-              let description = localize('metaDescriptionText').substring(0,120);
+              let description = localize('metaDescriptionText');
               if (responseData.suttaplex.blurb) {
-                  description = responseData.suttaplex.blurb.substring(0,120);
+                  description = responseData.suttaplex.blurb;
               }
-              const twitterDescription = description.substring(0,120);
               const title = responseData.translation ? responseData.translation.title : responseData.root_text.title;
               const author = responseData.translation ? responseData.translation.author : responseData.root_text.author;
-              return {
-                'og:title': `${title}—${author}`,
-                'og:description': description,
-                title: `${title}—${author}`,
-                description: description,
-                'twitter:title': `${title}—${author}`,
-                'twitter:description': twitterDescription
-              };
+
+              document.dispatchEvent(new CustomEvent('sherby-metadata', {
+                  detail: {
+                      title: `${title}—${author}`,
+                      description: description
+                  }
```

Comment:
the customEvent will probably need `bubbles: true, composed: true` next to `detail` to penetrate the DOM to index.html.